src/python/qequlacs/utils.py

```diff
@@ -11,8 +21,23 @@ def convert_circuit_to_qulacs(circuit):
 
 def append_gate(gate, qulacs_circuit):
     """
-    This function is based on the `_try_append_gate` function from https://github.com/qulacs/cirq-qulacs/blob/master/cirqqulacs/qulacs_simulator.py .
+    Appends corresponding Qulacs gate to QuantumCircuit.
+
+    This function is based on the `_try_append_gate` function from
+    https://github.com/qulacs/cirq-qulacs/blob/master/cirqqulacs/qulacs_simulator.py
     """
+
+    # From z-quantum.core _circuit.py: exponent=gate.params[0] / np.pi
+    # So gate.params[0] = exponent * np.pi
+    if gate.params:
+        angle = gate.params[0]
+    else:
+        angle = 0
+
+    modifiers = [] # dagger and so on
+    if hasattr(gate,  'modifiers'):
+        modifiers = gate.modifiers
```

Comment:
I'm assuming these are "Zapata" gates in which case I don't believe we use modifiers currently

Reply:
Right, these are gates from z-quantum-core,
Modifiers allow to get Sdag, Tdag gates.
If it is not planned to use - we may remove this.